### DISCUSSION TITLE
Is there any coding convention in mobile(flutter) code?

### DISCUSSION THREAD

**Discussion by xpwmaosldk** [immich-app/immich#3260] [Q&A] [answered]
I came across the following code:
```
loadDb() async {
//blah blah
}
```
This doesn't adhere to the naming conventions recommended in [Dart](https://dart.dev/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words), so it's somewhat visually jarring.
As there might be many instances like this, I plan to correct them according to the official documentation whenever I have time, provided there are no specific coding convention in Immich.

**Comment by alextran1502** [comment, marked as answer]
We don't have one unfortunately. So you are welcome to correct them accordingly to Dart recommendations
